### PR TITLE
Add models from last 4 months

### PR DESCRIPTION
**Pull request by Kim2091**
Adds all models (at least, that i could find) from the last 4 months. Also adds many architectures and users, as necessary. Also fixed a minor issue in sudo's cugan listing (sorry for not splitting this out)

### CODE REVIEW

**Review by RunDevelopment — COMMENTED** [review]
Could you please add more tags to these models? I already suggested a few, but it seems like a lot of models here are missing tags. Some don't have any at all.

File: data/models/1x-SwatKats-SPAN.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "license": "CC-BY-NC-SA-4.0",
+    "tags": [
+        "cartoon"
+    ],
+    "description": "This is SaurusX's original description:\nFix vertical blur / split lines / shadowing. A 1x model of my 2xSwatKats. Resolves the same video problems as before, but 1x and faster and meant for chaining to other 2x models (or whatever). Input MUST be 540 vertical as the blur problem is very resolution sensitive.\n\n-------\n\nThe goal for this retrain was to get the \"magic\" of SwatKatsLite into SPAN for much faster video processing. It's about 80% of the way there. There's some artifacts here and there that aren't present in the original, and it has small amounts of color halos in certain areas. I've retrained this model so many times that I've just decided to release it. This particular release was trained to ~450k.\n\nThe included Family Guy image shows the color ringing issue. Sorry. \n\nUpdate: I've added a second alt model that has less color ringing, but doesn't do quite as good with line repair. I believe I've reached architecture limits",
+    "date": "2023-12-15",
+    "architecture": "span",
+    "size": [
+        ""
+    ],
```
[the remark this answers is not in the document]
Comment (Kim2091) [reply data/models/1x-SwatKats-SPAN.json:13]: Oops, yeah. I was counting on the test to tell me if they were empty (just as it did for resources). I'll resolve these

File: data/models/1x-Bendel-Halftone.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{
+    "name": "Bendel_Halftone",
+    "author": "bendel",
+    "license": "CC-BY-4.0",
+    "tags": [
+        "restoration",
+        "text"
+    ],
```
[the remark this answers is not in the document]
Comment (Kim2091) [reply data/models/1x-Bendel-Halftone.json:8]: Actually it's for any scan from printed media. It's a very wide range, so I wasn't really sure what else to put other than restoration. Having a dehalftone tag would be nice

File: data/models/1x-Frankendata-FullDegradation-SRFormer.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+    "name": "Frankendata_FullDegradation_SRFormer",
+    "author": "crustaceous-d",
+    "license": null,
+    "tags": [
+        "photo",
+        "restoration"
+    ],
```
[the remark this answers is not in the document]
Comment (Kim2091) [reply data/models/1x-Frankendata-FullDegradation-SRFormer.json:8]: This is a 1x model, so I don't think that's quite accurate? It restores output from another upscaler however

**Comment by Kim2091** [comment]
Ready to be merged, unless I overlooked something

**Review by joeyballentine — APPROVED** [review]

**Pull request merged**